generator/testdata/rust/openapi/golden/src/model.rs

```diff
@@ -2733,6 +2734,7 @@ pub struct GetIamPolicyRequest {
     /// To learn which resources support conditions in their IAM policies, see the
     /// [IAM
     /// documentation](https://cloud.google.com/iam/help/conditions/resource-policies).
+    #[serde(rename = "options.requestedPolicyVersion")]
```

Comment:
This should be requested_policy_version, right?

Reply:
I am going to punt on this.  It is a query parameter, and not serialized at all.